### ISSUE TITLE
Shouldn't the exclude property be an array?

### ISSUE THREAD

**Issue by evanplaice** [MathieuLoutre/grunt-aws-s3#85]
It's pretty common to have more than one exclude.

Shouldn't this be an array, or is there some way to designate multiple exclude blobs at once?

**Comment by MathieuLoutre** [comment]
Yeah, I guess that makes sense and shouldn't be too hard. I don't think I'll have the time to look into it until this weekend but if you want to take a stab at it, I'll take a PR :)
